assets/promptflow/evaluators/models/coherence-evaluator/spec.yaml

```diff
@@ -5,8 +5,8 @@ properties:
   is-promptflow: true
   is-evaluator: true
   show-artifact: true
-  _default-display-file: ./evaluator/flow/prompt.jinja2
```

Comment:
I thought for the prompt based ones, we want to show its prompt to customers by default.

Reply:
I believe the decision to show articles or not, and what to display by default, was defined by Wenxin.

Let's keep the behavior the same as before. In general:

Prompt-based evaluators: show the prompt
Content safety evaluators: do not show the artifact
Math-based evaluators: show the code